src/modules/dome.wren

```diff
@@ -45,6 +45,10 @@ class Version {
 
 class Process {
   foreign static f_exit(n)
+  foreign static args
```

Comment:
This method should be `f_args`, since it's intended to be an internal/private property, rather than a user-facing one.

Reply:
I think both should be user facing since the idea is:

- args: Returns all arguments.
- arguments: Returns user only arguments ( args - DOME arguments - `./dome` )

Reply:
We might need to check if it does return all arguments. I think optparse might omit `--` if it's an argument on it's own.

Reply:
Having double-checked, what happens is that once `--` is encountered, everything after it is assumed not to be commandline options. This can lead to some weird behaviour if you haven't manually specified as the first non-option argument to be your main wren file. Tricky problem to resolve without changing the commandline usage slightly.

Reply:
I think is fine for now rename `args` to `f_args` since it will contain information normally not relevant to the final user.

and `arguments` would contain everything except `./dome`

if you pass a string to the arguments.  it would return the proper value if there is a match.

```js
static arguments { Process.f_args[1..-1] }

static arguments(needle) {
     var result = null
    Process.f_args.each { |item| 
          if (item.contains(needle)) {
                result = item
         }
   }
  return result
}
```